packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -200,6 +202,7 @@ function* sendTipAsync() {
   const { waitForRemoteConfig } = yield* getContext('remoteConfigInstance')
   const isNativeMobile = yield* getContext('isNativeMobile')
   yield call(waitForRemoteConfig)
+  yield* call(waitForBackendSetup)
```

Comment:
So do all these essentially wait until we've come online and then initialized backend to run?

Reply:
Yeah, that's right. The waitForAccount gate was opening since I added the optimistic offline account load. In the past, we could assume account meant backend was present too, but that's not true anymore.